apps/dash-study-browser/app.py

```diff
@@ -0,0 +1,322 @@
+                                    ],
+                                ),
+                                html.Div(
+                                    className="padding-top-bot",
+                                    children=[
+                                        html.H6("Choose the type of plot"),
+                                        dcc.RadioItems(
+                                            id="chart-type",
+                                            options=[
+                                                {"label": "Box Plot", "value": "box"},
+                                                {
+                                                    "label": "Violin Plot",
+                                                    "value": "violin",
+                                                },
+                                            ],
+                                            value="violin",
+                                            labelStyle={
+                                                "display": "inline-block",
+                                                "padding": "12px 12px 12px 0px",
+                                            },
+                                        ),
+                                    ],
+                                ),
+                                html.Div(
+                                    className="padding-top-bot",
+                                    children=[
+                                        html.H6("CSV File"),
+                                        dcc.Upload(
+                                            id="upload-data",
+                                            children=html.Div(
+                                                className="upload",
+                                                children=[
+                                                    html.P("Drag and Drop or "),
+                                                    html.A("Select Files"),
+                                                ],
+                                            ),
+                                        ),
+                                    ],
+                                ),
+                            ],
+                        )
+                    ],
+                ),
+                # Graph
+                html.Div(
+                    className="eight columns card-left",
+                    children=[
+                        html.Div(
+                            className="bg-white",
+                            children=[
+                                html.H5("Animal data plot"),
+                                dcc.Graph(id="plot"),
+                            ],
+                        )
+                    ],
+                ),
+            ],
+        ),
+    ]
+)
+
+# Callback to create error message
+@app.callback(Output("error-message", "children"), [Input("upload-data", "contents")])
```

Comment:
```suggestion
@app.callback(
    Output("error-message", "children"), 
    [Input("upload-data", "contents")])
```

Just for consistency with the other callback below!

Reply:
I think that was formatted by `black`